src/util/log.cc

```diff
@@ -322,7 +322,7 @@ logger::do_log(log_level level, log_writer& writer) {
     }
     auto print_once = [&] (internal::log_buf::inserter_iterator it) {
       if (local_engine) {
-          it = fmt::format_to(it, " [shard {:{}}]", this_shard_id(), _shard_field_width);
+          it = fmt::format_to(it, " [shard {:{}}/{}]", this_shard_id(), _shard_field_width, current_scheduling_group().name());
```

Comment:
Nowadays logs are eyes-friendly since messages' text starts at the same offset within the line, thanks to this `_shard_field_width` thing. Adding the sched group label at the tail breaks this nicety :(

Reply:
Yes it does. We could reserve some space for the scheduling group name.

Reply:
can we have a short name or just the id instead the full name in the logging message? because the names of scheduling groups are quite lengthy, like "background_reclaim", "streaming", "compaction"? we can have a shorten function which generate a signature like "br" from "background_reclaim", if there are no underscores in the fullname, the first three characters are used, so the short name of "streaming" would be "str", if we are good on this, i will work on adding this to `task_queue`.

Reply:
A short name is a good idea.

Reply:
#1666